README.md

```diff
@@ -1,66 +1,76 @@
-
-The main functionality comes from these modules:
-
-* [SilverStripe Composer Security Checker](https://github.com/bringyourownideas/silverstripe-composer-security-checker "Check your SilverStripe application for security issues")
-* [SilverStripe Composer Update Checker](https://github.com/bringyourownideas/silverstripe-composer-update-checker "Check your SilverStripe application for available updates of dependencies.")
-* [SilverStripe Composer Versions](https://github.com/bringyourownideas/silverstripe-composer-versions "Provides your installed composer versions within your SilverStripe app, for review or other use cases.")
-
-
-## Requirements and installation
-
-### Requirements
+## Requirements
 
 * You require the composer.json and composer.lock files to be available and readible in the environment you plan to use this module. All information is based on these files.
-* Install at least one of the modules mentioned under "Source of the information". As a development dependency should be fine in most cases.
+* Install at least one of the modules mentioned under "Source of the information". A development dependency should be fine in most cases.
 * The queuedjob module is a dependency as the checks are scheduled using queuedjobs. This saves you time and work at the end.
 
+Note: Release line 1 is compatible with SilverStripe 3. For SilverStripe 4, please see the 2.x release line.
 
-### Installation
+### Suggested Modules
 
-Run the following commands to install the package including all suggestions and populate the information initially:
+While the installation of the following modules is optional, it is recommended:
+- [bringyourownideas/silverstripe-composer-security-checker](https://github.com/bringyourownideas/silverstripe-composer-security-checker) checks for known security vulnerabilities
+- [bringyourownideas/silverstripe-composer-update-checker](https://github.com/bringyourownideas/silverstripe-composer-update-checker) checks for available updates of dependencies
+     
 
+### Installation 
+ 
+Install the maintenance package.
 ```
-# install the packages
 composer require bringyourownideas/silverstripe-maintenance
-composer require bringyourownideas/silverstripe-composer-security-checker
-composer require bringyourownideas/silverstripe-composer-update-checker
-composer require bringyourownideas/silverstripe-composer-versions
-
-# schedule the population of the data
-php ./framework/cli-script.php dev/build
+```
 
-# run the queuedjobs
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
-php ./framework/cli-script.php dev/tasks/ProcessJobQueueTask
+Build schema and queue a job to populate the database:
 ```
+sake dev/build
+```
+ 
+Build after installing:
+```
+sake dev/build
+```
```

Comment:
This looks like it's duplicated now =) 

dev/build no longer populates the database in the context of this module